src/Angelica/kata31_10/Capicua.py

```diff
@@ -0,0 +1,19 @@
+def is_capicua(value):
+    word = str(value)
+
+    index = -1
+    same = 0
+
+
+    for x in range(0,len(word)/2):
+
+        if word[x] == word[index]:
+            same = same+1
+        index = index - 1
+
+    if same == (len(word) / 2):
```

Comment:
You can refactor the if statement as following: `return same == (len(word) / 2)`